src/modules/jobs/models/tech-load.model.ts

```diff
@@ -0,0 +1,399 @@
+  }
+}
+
+interface JobData {
+  accountNumber: number,
+  firstName: string,
+  lastName: string,
+  location: Location,
+  appointment: string,
+  email: string,
+  phone: number,
+  jobType: string,
+  transportType: string,
+  history: History[],
+  facilities: Facility[],
+  services: Services,
+}
+
+interface Facility {
+  heading: string,
+  address: string,
+  cable?: number,
+  pair?: number,
+  port?: number,
+}
+
+interface Services {
+  equipment: string[];
+  internetSpeed?: number;
+  potsNumber?: number;
```

Comment:
For reference, the API responses we get, numbers like these are typically strings.

Reply:
What's the benefit to that over having them as numbers?

Reply:
Still wanting to know @looksliketyler